RTClib.cpp

```diff
@@ -1559,39 +1560,14 @@ void RTC_PCF8563::setAlarm(const DateTime &dt, Pcf8563AlarmMode alarm_mode) {
 /*!
     @author Leonardo Bispo
     @brief  Set Weekly Alarm values mode on the PCF8563
-    @param dow  Day of the week to trigger the alarm
+    @param dow  Day of the week to trigger the alarm, Sunday is 0
```

Comment:
“Sunday is 0” is misleading. If the user provides `0` (or any other integer) as the first argument, the overload will be resolved by the compiler to the method `RTC_PCF8563::setAlarm(uint8_t day, uint8_t hour, uint8_t min)`.

Reply:
6e593953ab9cf0ddb627eaa577a4070afe9f53f3